sessions/stats-analyses-basic.qmd

```diff
@@ -134,52 +172,36 @@ graph LR
-uncertainty in our observations (the data) and in the model. And the
-basic simplified math of a statistical model looks mostly the same:
-
-$$y = intercept + x + error $$
-
-So if we use the example of plant growth, it would look like:
-
-$$Growth = Sunlight + Water$$
-
-It's a bit more complicated than this, but this is enough to describe it
-for this course. Throughout the rest of the session, we use specific
-terms to describe each item in this formula. "Outcome" (also called
-"dependent variable") refers to the $y$, "predictor" (or "independent
-variable") refers to the $x$. The error and intercept are calculated for
-us when we fit the model to the data. The intercept is when x is equal
-to zero (the "y-intercept" on a plot). The error is the difference
-between what the model estimates and the real value. It plays a role in
-quantifying the model's uncertainty.
-
-Considering the mathematical nature of statistical models, there is also
-a logic and "workflow" to making these models!
-
-1.  Write a research question, designed in a way that might look like
-    the diagram above. Usually this step needs to be revisited, revising
-    the question after trying to construct the theoretical model, that
-    describes the measurable (and unmeasurable) parameters in the model,
-    and vice versa.
-2.  Based on the model and the type of measured parameters used
-    (continuous or binary), select the best mathematical model "type".
-    Nearly all models in statistics start from the base of a linear
-    regression (e.g. ANOVA is a special form of regression, t-test is a
-    simpler version of ANOVA), so the model "type" will probably be a
-    form of regression.
-3.  Measure your parameters (in the plant growth example, that might be
-    the amount of water given in liters per day, amount of plant growth
-    in weight, and amount of sunlight in hrs per day). Usually, this
-    measured data need to be processed in a special way to fit the
-    specifics of the model, research question, and practices of the
-    field.
-4.  Fit the data to the theoretical model in order to estimate the
-    values ("coefficients") of the model parameters as well as the
-    uncertainty in those values.
-5.  Extract the values and their uncertainty from the model and present
-    them in relation to your research questions.
+or mathematically:
+
+$$growth = sunlight + water$$
+
+This expression is known as our theoretical model. However, not all
+theoretical models can be tested against the real world. We need to
+structure the theoretical model so that we can measure the parameters
+involved. In this case, we need to measure plant growth (e.g., weight in
+grams), the amount of water given per day (in liters), and the number of
+sunlight hours per day.
+
+This also means that when formulating a research question, there is an
+order, or workflow, to follow:
+
+### Workflow for Statistical Models
+
```

Comment:
```suggestion
```

Since the next list is connected directly to the paragraph above, inserting a header here would break that connection, so I removed it :relaxed: